teams/views.py

```diff
@@ -31,7 +31,7 @@ def change_teams(request, hackathon_id):
         teams = []
         participants_still_to_distribute = []
     elif len(hackathon.teams.all()) == 0:
-        teamsize = 3
+        teamsize = hackathon.teamsize
```

Comment:
`team_size` --- even matches below for `team_sizes` plural

Reply:
Updated to `team_size`